projects/ElVentana/ElVentana.cfg

```diff
@@ -0,0 +1,282 @@
+propagate_field = 1
+propagate_vlasov_acceleration = 1
+propagate_vlasov_translation = 1
+project = ElVentana
+ParticlePopulations = electron
+ParticlePopulations = proton
+dynamic_timestep = 0
+
+#[restart]
+#filename = restart.0000000.2018-10-23_05-20-32.vlsv
+
+[io]
+diagnostic_write_interval = 1
+write_initial_state = 0
+restart_walltime_interval = 41400
+number_of_restarts = 2
+
+system_write_t_interval = 0.0001
+system_write_file_name = bulk
+system_write_distribution_stride = 0
+system_write_distribution_xline_stride = 0
+system_write_distribution_yline_stride = 0
+system_write_distribution_zline_stride = 0
+
+
+system_write_t_interval = 0.0005
+system_write_file_name = distributions
+system_write_distribution_stride = 0
+system_write_distribution_xline_stride = 10
+system_write_distribution_yline_stride = 1
+system_write_distribution_zline_stride = 10
+
+#system_write_t_interval = 500
+#system_write_file_name = fullf
+#system_write_distribution_stride = 1
+#system_write_distribution_xline_stride = 0
+#system_write_distribution_yline_stride = 0
+#system_write_distribution_zline_stride = 0
+write_restart_stripe_factor = 24
+
+[bailout]
+min_dt = 1E-8
+max_memory = 60
+
+[gridbuilder]
+x_length = 1710
+y_length = 1
+z_length = 1060
+x_min = -380.0e6
+x_max = 133.0e6
+y_min = -1.5e5
+y_max = 1.5e5
+z_min = -159.0e6
+z_max = 159.0e6
+
+#timestep_max = 10000000
+dt = 1.e-6
+t_max = 0.03
+# dt = 0.0
+
+[proton_properties]
+mass = 1
+mass_units = PROTON
+charge = 1
+propagate_population = 1
+test_population = 0
+
+[proton_vspace]
+vx_min = -4.0e6
+vx_max = +4.0e6
+vy_min = -4.0e6
+vy_max = +4.0e6
+vz_min = -4.0e6
+vz_max = +4.0e6
+vx_length = 30
+vy_length = 30
+vz_length = 30
+
+[proton_sparse]
+minValue = 1.0e-15
+#minValue = 0e-14
+
+[proton_Magnetosphere]
+T = 0.5e6
+
+[electron_Magnetosphere]
+T = 0.5e6
+
+[electron_properties]
+mass = 1
+mass_units = ELECTRON
+charge = -1
+test_population = 1
+
+[electron_vspace]
+vx_min = -2.0e8
+vx_max = +2.0e8
+vy_min = -2.0e8
+vy_max = +2.0e8
+vz_min = -2.0e8
+vz_max = +2.0e8
+vx_length = 40
+vy_length = 40
+vz_length = 40
+
+[electron_sparse]
+minValue = 1.0e-20
+#dynamicAlgorithm = 1
+#dynamicMinValue1 = 1.e-21
+#dynamicMinValue2 = 1.e-20
+#dynamicBulkValue1 = 1.e4
+#dynamicBulkValue2 = 1.e6
+
+[boundaries]
+periodic_x = no 
+periodic_y = yes
+periodic_z = no
+boundary = Outflow
+boundary = Maxwellian
+boundary = Ionosphere
+
+[ionosphere]
+centerX = 0.0
+centerY = 0.0
+centerZ = 0.0
+geometry = 2
+radius = 30.0e6
+
+[proton_ionosphere]
+rho = 1.0e6
+VX0 = 0
+VY0 = 0
+VZ0 = 0
+taperRadius = 10.0e6
```

Comment:
Can be dropped but doesn't hurt either. Same for electrons.

Reply:
Actually support for the ionosphere is still in there. Not strictly necessary I guess but ensures that values inside the inner boundary don't go crazy.

Reply:
I meant just taperRadius, as it's smaller than ionosphere radius it's the same as not setting it at all I think.